irods/test/scripts/test007_pam_features_in_new_auth_framework.bats

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bats
+#
+# Test creation of .irodsA for iRODS pam_password authentication, this time purely internal to the PRC
+# library code.
+
+. "$BATS_TEST_DIRNAME"/test_support_functions
+PYTHON=python3
+
+# Setup/prerequisites are same as for login_auth_test.
+# Run as ubuntu user with sudo; python_irodsclient must be installed (in either ~/.local or a virtualenv)
+#
+
+ALICES_ORIGINAL_PAM_PASSWORD=test123
+ALICES_NEW_PASSWORD=test_1234
+
+setup()
+{
+    export SKIP_IINIT_FOR_PASSWORD=1
```

Comment:
This looks new? Does it change the behavior of the script on L19? If so, is that documented somewhere?

Reply:
Not yet documented.  Could do that in the `test_support_functions` ....

Reply:
Yes, please do.

Reply:
OK. Note that this change will not be seen in this pull request, but rather in my dev branch `502-reb-000.SQ` .

Reply:
> OK. Note that this change will appear in #508 rather than this one.

For proof of which ref [this comment](https://github.com/irods/python-irodsclient/pull/508#issuecomment-2723576269) and the surrounding commits. 

Therefore I'm now resolving!